packages/models-library/tests/test_docker.py

```diff
@@ -12,6 +14,8 @@
 )
 from pydantic import ValidationError, parse_obj_as
 
+_faker = Faker()
```

Comment:
Sometimes you use `faker` and sometimes `_faker` (lines 17 and 9), could you provide some context for my learning here please? thx!

Reply:
actually we have a faker fixture. so there is I think absolutely no need to use ```_faker``` construct.

Reply:
When you try to generate some defaults at a module level context, you do not have access to the object provided via the fixture. Here I create an instance to use Faker at module level.

Doing it differently it would only overcomplicate things